src/cls/rgw/cls_rgw_types.cc

```diff
@@ -405,6 +405,15 @@ bool rgw_cls_bi_entry::get_info(cls_rgw_obj_key *key,
   return account;
 }
 
+void rgw_cls_bi_entry::generate_test_instances(list<rgw_cls_bi_entry*>& o)
+{
+  rgw_cls_bi_entry *m = new rgw_cls_bi_entry;
+  m->type = BIIndexType::Plain;
+  m->idx = "idx";
+  o.push_back(m);
+  o.push_back(new rgw_cls_bi_entry);
```

Comment:
**ceph::buffer::list** is not initalized in **generate_test_instances function.** I don't know how to put value in **ceph::buffer::list**.